code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -11,7 +11,7 @@
 	var/translate_hive = 0
 	var/syndie = 0
 	var/list/channels = list()
-	matter = list(MATERIAL_STEEL = 1)
+	matter = list(MATERIAL_STEEL = 1, MATERIAL_SILVER = 1)
```

Comment:
Odd change. I know you're doing this so that somebody can't mass-produce encryption keys, but now somebody can extract silver from existing encryption keys.

Reply:
Not like silver is THAT uncommon. You can gain full stacks by gathering and recycling cheap powercells, not like gathering headset keys (which is less common than powercells) for silver would change anything. However, if you insist - i will remove that right now in a second.